modules/accounts/accounts.js

```diff
@@ -0,0 +1,233 @@
+const confFile = __dirname + '/../../config.json';
+const jsonfile = require('jsonfile');
+var eztz = {};
+var config = jsonfile.readFileSync(confFile);
+
+const { Helper } = require('../../helper');
+
+class Accounts{
+
+    confFile = __dirname + '/../../config.json';
```

Comment:
Why are you reimporting the same variables ( confFile, jsonfile, config) again. You can create a separate module to handle reading and writing to the file or at least handling the common functionality from accounts.js, transactions.js. That way code becomes easier to maintain.